tests/test_utils.py

```diff
@@ -440,18 +440,87 @@ def delete_config_from_filesystem(
     )
 
 
+def get_snowflake_connection_url() -> str:
+    """Get snowflake connection url from environment variables.
+
+    Returns:
+        String of the snowflake connection url.
+    """
+    sfAccount = os.environ.get("SNOWFLAKE_ACCOUNT")
+    sfUser = os.environ.get("SNOWFLAKE_USER")
+    sfPswd = os.environ.get("SNOWFLAKE_PW")
+    sfDatabase = os.environ.get("SNOWFLAKE_DATABASE")
+    sfSchema = os.environ.get("SNOWFLAKE_SCHEMA")
+    sfWarehouse = os.environ.get("SNOWFLAKE_WAREHOUSE")
+
+    return f"snowflake://{sfUser}:{sfPswd}@{sfAccount}/{sfDatabase}/{sfSchema}?warehouse={sfWarehouse}"
```

Comment:
I know that getting the environment set up for BigQuery and Snowflake can be pretty hairy..were you able to get these running on Azure?